service/controller/v10/version_bundle.go

```diff
@@ -145,6 +145,6 @@ func VersionBundle() versionbundle.Bundle {
 		Name:         "aws-operator",
 		Time:         time.Date(2018, time.March, 22, 15, 47, 0, 0, time.UTC),
```

Comment:
Please change date of release.

We already voted about that and most of the people agreed to use date of release.

https://gigantic.slack.com/archives/C04TGHDEF/p1525071756000249